storage/remote/influxdb/client.go

```diff
@@ -14,148 +14,97 @@
 package influxdb
 
 import (
-	"bytes"
-	"encoding/json"
-	"fmt"
-	"io/ioutil"
 	"math"
-	"net/http"
-	"net/url"
-	"time"
 
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/model"
 	"github.com/prometheus/log"
 
-	"github.com/prometheus/prometheus/util/httputil"
-)
-
-const (
-	writeEndpoint   = "/write"
-	contentTypeJSON = "application/json"
+	idb_client "github.com/influxdb/influxdb/client"
```

Comment:
Neither camelCase nor under_scores are common for package names in Go in general, but for renamed imports we already use underscores for this at least here: https://github.com/prometheus/prometheus/blob/master/storage/local/index/leveldb.go#L20-L23